rollup.config.js

```diff
@@ -1,22 +1,23 @@
-import path from 'path'
-import pkg from './package.json'
+const entries = ['src/index.js', 'src/matchers.js']
 
 export default [
   {
-    input: {
-      index: 'src/index.js',
-      matchers: 'src/matchers.js',
+    input: entries,
+    output: {
+      dir: 'dist',
+      entryFileNames: '[name].mjs',
```

Comment:
You can add `chunkFileNames: '[name]-[hash].mjs'`, so chunks will also have correct filepath.

You can also add `preserveModules: true`, then rollup will not generate "chunks", it will have the same file structure as your source code. 

I guess the first solution is closest to how it was, so 😄

Reply:
Good eye, thanks for catching that.  I've taken your first suggestion, and mirrored it on the .js as well, just for consistency / to be explicit.